app/screens/global_drafts/components/tabbed_contents/draftTabsHeader.tsx

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useMemo} from 'react';
+import {FormattedMessage} from 'react-intl';
+import {Pressable, Text, View} from 'react-native';
+
+import Badge from '@components/badge';
+import {useTheme} from '@context/theme';
+import {DRAFT_SCREEN_TAB_SCHEDULED_POSTS, type DraftScreenTab} from '@screens/global_drafts';
+import {TAB_CONTAINER_HEIGHT} from '@screens/global_drafts/contants';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+const DRAFT_TAB_INDEX = 0;
+const SCHEDULED_POSTS_TAB_INDEX = 1;
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
+    return {
+        tabContainer: {
+            display: 'flex',
+            flexDirection: 'row',
+            height: TAB_CONTAINER_HEIGHT,
+        },
+        tab: {
+            width: '50%',
+            flex: 1,
+            flexDirection: 'row',
+            position: 'relative',
+            height: 44,
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center',
+            gap: 6,
+            borderBottomWidth: 2,
+            borderColor: 'transparent',
+            borderBottomColor: changeOpacity(theme.centerChannelColor, 0.08),
+        },
+        activeTab: {
+            borderBottomColor: theme.buttonBg,
+        },
+        badgeStyles: {
+            position: 'relative',
+            color: changeOpacity(theme.centerChannelColor, 0.75),
+            backgroundColor: changeOpacity(theme.centerChannelColor, 0.08),
+            alignSelf: 'center',
+            left: 0,
+        },
+        activeBadgeStyles: {
+            color: theme.buttonBg,
+            backgroundColor: changeOpacity(theme.buttonBg, 0.08),
+        },
+        tabItemText: {
+            ...typography('Body', 100, 'SemiBold'),
+            color: changeOpacity(theme.centerChannelColor, 0.75),
+        },
+        activeTabItemText: {
+            color: theme.buttonBg,
+        },
+    };
+});
+
+type Props = {
+    draftsCount: number;
+    scheduledPostCount: number;
+    selectedTab: DraftScreenTab;
+    onTabChange: (tab: DraftScreenTab) => void;
+}
+
+export function DraftTabsHeader({draftsCount, scheduledPostCount, selectedTab, onTabChange}: Props) {
+    const theme = useTheme();
+    const styles = getStyleSheet(theme);
+
+    const draftCountBadge = useMemo(() => {
+        // eslint-disable-next-line no-warning-comments
+        // TODO: when integrating with database, handle here the case of draft count being 0, and return undefined to show no badge
```

Comment:
Can we implement this todo? If draft or scheduled post is 0, there should be no badge in the tab. Currently, I believe it will show a badge with count 0.

Reply:
Done.